setup.sh

```diff
@@ -12,7 +12,7 @@ echo "-------------------------------------------"
 echo "Installing bb8 user and group"
 
 if ! id -u bb8 > /dev/null 2>&1; then
-    useradd bb8 -d /var/lib/bb8
+    useradd bb8 -U -d /var/lib/bb8
```

Comment:
looks like this `-U` flag means the user is added to newly created group of the same name
https://linux.die.net/man/8/useradd

Reply:
* [`2018-009-i1397-201801rft-add-yf-candidate3`](2018-009-i1397-201801rft-add-yf-candidate3) "Add RFP YF candidate3"